packages/agent-sdk-ts/docs/python-parity.md

```
@@ -0,0 +1,273 @@
+      +name
+      +description
+      +keywords/tasks/alwaysOn
+      +aliases
+      +validateInput()
+    }
+    AgentContextTS --> SkillTS
+```
+
+## Event logging, persistence, and events
+
+**EventLog/persistence**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)